specification/translation/data-plane/Azure.AI.DocumentTranslation/readme.md

```diff
@@ -1,47 +1,39 @@
-# Cognitive Services Translation Document SDK
+# Cognitive Services Document Translation  SDK
 
 > see https://aka.ms/autorest
 
-Configuration for generating Translation Document SDK.
+Configuration for generating Document Translation  SDK.
 
-## Releases
+### Basic Information
+
+These are the global settings for the app.
+The current release is `2024-05-01`.
 
-The current release of Document is **2023-11-01-preview**.
 ``` yaml
-tag: 2023-11-01-preview
+tag: 2024-05-01
 add-credentials: true
-clear-output-folder: true
 openapi-type: data-plane
 ```
 
+# Releases
+
+### Release Tag: 2024-05-01
+These settings apply only when `--tag=2024-05-01` is specified on the command line.
+
+``` yaml $(tag) == '2024-05-01'
+input-file: 
+  - stable/2024-05-01/openapi.json
```

Comment:
Seems like the swagger apiview step isnt finding this file. You might have to add the previous folder section to the path

Reply:
```suggestion
  - Azure.AI.DocumentTranslation/stable/2024-05-01/openapi.json
```

Reply:
You probably have to make the same change for the preview tag since CI didnt find that one either.

Reply:
Updated